Sources/PrinceOfVersions/PoVDataTypes/Version.swift

```diff
@@ -31,7 +31,30 @@ public class Version: NSObject {
         UserDefaults.standard.set(true, forKey: versionUserDefaultKey)
     }
 
+    required public init(from decoder: Decoder) throws {
+        major = 0
+        minor = 0
```

Comment:
You can put default values for those as well instead of doing this 😅 
But I would go with making every ver in this class as a let 😉